cognee/api/v1/permissions/routers/get_permissions_router.py

```
@@ -2,45 +1,63 @@
 
         if not user:
             raise UserNotFoundError
         elif not group:
             raise GroupNotFoundError
 
-        user.groups.append(group)
+        try:
+            # Add association directly to the association table
+            stmt = insert(UserGroup).values(user_id=user_id, group_id=group_id)
+            await db.session.execute(stmt)
+        except IntegrityError as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)